src/flows/parser_item_html_text_backfill_file_flow.py

```diff
@@ -1,18 +1,122 @@
+STAGE_CHUNK_ROWS = 10000
+
+LOAD_SQL = f"""
+copy into snapshot.item.article_content_v2
+(resolved_id, html, text)
+from %(uri)s
+storage_integration = aws_integration_readonly_prod
+file_format = (type = 'CSV', skip_header=1, FIELD_OPTIONALLY_ENCLOSED_BY='"')
+on_error=ABORT_STATEMENT;
+"""
+
+@task()
+def extract(key: str) -> pd.DataFrame:
+    logger = prefect.context.get("logger")
+    logger.info(f"Extracting file: {str(key)}")
+    bucket = SOURCE_S3_BUCKET
+    s3 = boto3.resource('s3')
+    response = s3.Object(bucket, key).get()
+    df = pd.read_csv(response['Body'], escapechar='\\', sep="|", header=None, usecols=[0,1],
+                     names=['resolved_id', 'compressed_html'])
+    return df
+
+
+@task()
+def transform(df: pd.DataFrame) -> pd.DataFrame:
+    df['html'] = [zlib.decompress(base64.b64decode(compressed_html)).decode() for compressed_html in df['compressed_html']]
+    df.drop(columns=['compressed_html'], inplace=True)
+    df['text'] = [get_text_from_html(html) for html in df['html']]
+    return df
+
+
+
+def get_stage_prefix() -> str:
+    """
+    The stage prefix is the destination for the transformed stage files used for Snowflake load
+    :return: S3 Stage prefix for each run instance.
+    """
+    s3_prefix = STAGE_S3_PREFIX
+    flow_run_id = prefect.context.get('flow_run_id')
+    return f"{s3_prefix}{flow_run_id}"
+
+def stage_chunk(index: int, df: pd.DataFrame) -> str:
+    bucket = STAGE_S3_BUCKET
+    s3_prefix = get_stage_prefix()
+    key = f"{s3_prefix}/{index}.csv.gz"
+
+    csv_buffer = BytesIO()
+    with gzip.GzipFile(mode='w', fileobj=csv_buffer) as gz_file:
+        df.to_csv(gz_file, index=False)
+
+    s3 = boto3.resource('s3')
+    s3.Object(bucket, key).put(Body=csv_buffer.getvalue())
+    return key
+
+
+@task()
+def stage(df: pd.DataFrame) -> [str]:
+    logger = prefect.context.get("logger")
+    chunks = [(i, df[i:i + STAGE_CHUNK_ROWS]) for i in range(0, df.shape[0], STAGE_CHUNK_ROWS)]
+    logger.info(f"Number of chunks created: {len(chunks)}.")
+    keys = [stage_chunk(i, chunk) for i, chunk in chunks]
+    logger.info(f"Staged keys: {*keys,}")
+    return keys
+
+@task()
+def load(key: str) -> str:
+    uri = f"s3://{STAGE_S3_BUCKET}/{key}"
+    logger = prefect.context.get("logger")
+    logger.info(f"Snowflake loading key: {uri}")
+    SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(data={'uri': uri}, query=LOAD_SQL)
+    return key
+
+@task()
+def cleanup(key: str):
+    bucket = STAGE_S3_BUCKET
+    logger = prefect.context.get("logger")
+    logger.info(f"deleting file: {str(key)}")
+    s3 = boto3.resource('s3')
+    s3.Object(bucket, key).delete()
+
+@task
+def print_results(param_name, param_value):
+    print(f'{param_name}: {param_value}')
 
 
 with Flow(FLOW_NAME, executor=LocalDaskExecutor()) as flow:
     key = Parameter('key')
-    flow.logger.info(f"Processing key: {key}")
+    print_results('Processing key', key, task_args=dict(name="print_parameter_key"))
```

Comment:
We can drop this for the `Extracting file` logger statement in the `extract` function.